### ISSUE TITLE
[BUG]: Status bar for downloading resen-core images

### ISSUE THREAD

**Issue by ljlamarche** [EarthCubeInGeo/resen#63]
Downloading resen-core images can take a few minutes, so a status bar is displayed so the user can see the progress.  Instead of updating a single line, a new progress bar is printed on a new line at each updating, resulting in something like this:
```
[=====>                                             ]  11.99 %, 0.
[======>                                            ]  12.38 %, 0.
[======>                                            ]  12.84 %, 0.
[======>                                            ]  13.29 %, 0.
[======>                                            ]  13.74 %, 0.
[=======>                                           ]  14.17 %, 0.
[=======>                                           ]  14.62 %, 0.
[=======>                                           ]  14.95 %, 0.
[=======>                                           ]  15.24 %, 0.
[=======>                                           ]  15.67 %, 0.
[========>                                          ]  16.00 %, 0.
[========>                                          ]  16.45 %, 0.
[========>                                          ]  17.05 %, 0.
[========>                                          ]  17.58 %, 0.
[=========>                                         ]  18.11 %, 0.
[=========>                                         ]  18.58 %, 0.
[=========>                                         ]  19.03 %, 0.
[=========>                                         ]  19.46 %, 0.
[=========>                                         ]  19.87 %, 0.
[=========>                                         ]  19.80 %, 0.
[==========>                                        ]  20.27 %, 0.
[==========>                                        ]  20.67 %, 0.
[==========>                                        ]  21.13 %, 0.
[==========>                                        ]  21.51 %, 0.
[==========>                                        ]  21.95 %, 0.
```
The resen-core image will download successfully and this doesn't seem to impact any of the basic functionality, but it certainly looks weird.

**Comment by asreimer** [comment]
I don't see this problem in linux. Is this a macOS specific problem?

Does #65 fix this issue @ljlamarche?

**Comment by ljlamarche** [comment]
Yes, this is solved with #65.  Closing this now.
